aclgen.py

```diff
@@ -434,7 +434,11 @@ def WriteFiles(write_files):
     output.flush()
 
 
-def main(_):
+def main(args):
```

Comment:
There is no need for this section. Tests would pass if you did the FLAGS.Reset() in the setup or tear down phase in the tests.